src/lib/modules/charts/components/model-modal/model-modal.component.ts

```diff
@@ -13,25 +21,24 @@ import { ModalOptions } from 'ngx-bootstrap/modal';
         <ccc-model-modal
```

Comment:
the first bullet point above needs update, too "Requires input for selected dataset"